lib/rufo/command.rb

```diff
@@ -176,12 +189,16 @@ def self.parse_options(argv)
         filename_for_dot_rufo = value
       end
 
+      opts.on("-x", "--no-exit-code", "Return 1 in the case of failure, else 0") do
```

Comment:
What about `--simple-exit`? It's not like this flag causes there to be no exit code, that would be very impressive :P.